page-objects/pages/AllArticles.js

```diff
@@ -1,57 +1,46 @@
-class AllArticles{
-    get first()
-    {
-        return $$('.c-link__cover')[0]
-    }
-
-    get second()
-    {
-        return $$('.c-link__cover')[1]
-    }
-
-    get third()
-    {
-        return $$('.c-link__cover')[2]
-    }
-
-    get fourth()
-    {
-        return $$('.c-link__cover')[3]
-    }
-
-    get fifth()
-    {
-        return $$('.c-link__cover')[4]
-    }
-
-    get sixth()
-    {
-        return $$('.c-link__cover')[5]
-    }
-
-    get seventh()
-    {
-        return $$('.c-link__cover')[6]
-    }
-
-    get eighth()
-    {
-        return $$('.c-link__cover')[7]
-    }
-
-    get ninth()
-    {
-        return $$('.c-link__cover')[8]
-    }
-
-    get tenth()
-    {
-        return $$('.c-link__cover')[9]
-    }
-
-    get galleryCategory(){
-        return $('.qa-gallery-category')
-    }
-
+class AllArticles {
+  get first() {
```

Comment:
The name of this variable is not so suggestive. This comment is valid for all the other selectors from this file. The name of the variable should be suggestive when we read the name to understand what suppose to do and contain that variable.